src/main.rs

```diff
@@ -143,6 +138,65 @@ fn configure_color(cli_choice: Option<clap::ColorChoice>) {
     choice.write_global();
 }
 
+fn print_issue_url(config: &mut GlobalConfig) {
+    use bugreport::{bugreport, collector::*, format::Markdown};
+    let other_bug_url: &str = "https://github.com/obi1kenobi/cargo-semver-checks/issues/new?labels=C-bug&template=3-bug-report.yml";
+
+    let bug_report = bugreport!()
+        .info(SoftwareVersion::default())
+        .info(OperatingSystem::default())
+        .info(CommandLine::default())
+        .info(CommandOutput::new("cargo version", "cargo", &["-V"]))
+        .info(CompileTimeInformation::default())
+        .format::<Markdown>();
+
+    writeln!(
+        config.stdout(),
+        "{}System information:{}\n--------------------------\n{bug_report}",
+        Style::new()
+            .bold()
+            .fg_color(Some(Color::Ansi(AnsiColor::Cyan))),
+        Reset
+    )
+    .expect("Failed to print bug report system information to stdout");
+
+    let bug_report_url = urlencoding::encode(&bug_report);
+
+    let cargo_config = match Config::load() {
+        Ok(c) => toml::to_string(&c).unwrap_or_else(|s| {
+            eprintln!("Error serializing cargo build configuration: {}", s);
+            String::default()
+        }),
+        Err(e) => {
+            eprintln!("Error loading cargo build configuration: {}", e);
```

Comment:
The `eprintln!()` should probably be `writeln!(config.stderr(), ...)` now that we're plumbing the config through.